tests/test_cache.py

```diff
@@ -282,3 +419,19 @@ def get_cache_expire_string_year(self):
     @Cache(key="test_cache_expire_string_year", invalidate=True)
     def clear_cache_expire_string_year(self):
         pass
+
+    @Cache(key="test_cache_empty", expire=timedelta(seconds=2))
+    def get_cache_empty(self, param):
+        return param
+
+    @Cache(key="test_cache_empty", invalidate=True)
+    def clear_cache_empty(self):
+        pass
+
+    @Cache(key="test_cache_empty_false", cache_empty_result=False, expire=timedelta(seconds=2))
+    def get_cache_empty_false(self, param):

```

Comment:
the name `get_cache_empty_false` is not clear. 
suggestion:
`get_dont_cache_empty_results`